doc/sai-ptf/fdb_test_plan.md

```diff
@@ -0,0 +1,288 @@
+### Testing Objective <!-- omit in toc -->
+- test_port_age: Verifying if the dynamic FDB entry associated with the port is removed after the aging interval.
+- test_aging_after_move: Verifying the aging time refreshed if dynamic FDB entry associated with one port and then moved to another port (not the initial learning time)
+- test_mac_moving_after_aging: Verifying the mac can be learnt again after the mac aging reached.
+
+
+### Test steps: <!-- omit in toc --> 
+- test_port_age
+
+1. Set FDB aging time=10
+2. Create Packet with SMAC=``MacX`` DMAC=``Port1 MAC`` 
+3. Send packet on port2
+4. verify only receive a packet on port1
+5. Create a packet with DMAC=``MacX``
+6. Send packet on port1
+7. Verify only receive a packet on port2
+8. Wait for the ``aging`` time
+9. Send packet on port1
+10. Verify flooding packet to VLAN10 ports, except port1
+
+- test_aging_after_move
+
+1. Set FDB aging time=10
+2. Create Packet with SMAC=``MacX`` DMAC=``Port1 MAC`` 
+3. Send packet on port2
+4. verify only receive a packet on port1
+5. Create a packet with DMAC=``MacX``
+6. Send packet on port1
+7. Verify only receive a packet on port2
+8. Send packet on port3
+7. Verify only receive a packet on port2
+9. Wait for the ``aging`` time
+10. Send packet on port3
+11. Verify flooding packet to VLAN10 ports, except port1
+
+- test_mac_moving_after_aging
+
+1. Set FDB aging time=10
+2. Create Packet with SMAC=``MacX`` DMAC=``Port1 MAC`` 
+3. Send packet on port2
+4. verify only receive a packet on port1
+5. Create a packet with DMAC=``MacX``
+6. Send packet on port1
+7. Verify only receive a packet on port2
+8. Wait for the ``aging`` time
+9. Create Packet with SMAC=``MacX`` DMAC=``Port1 MAC`` 
+10. Send packet on port3(mac moved from Port2 to Port3)
+11. verify only receive a packet on port1
+12. Create and send a packet with DMAC=``MacX``
+13. Verify only receive a packet on port3
+
+## Test Group3: FDB flush
+### Case1: test_flush_vlan_static
+### Case2: test_flush_vlan_dynamic
+### Case3: test_flush_port_static
+### Case4: test_flush_port_dynamic
+### Case5: test_flush_all_static
+### Case6: test_flush_all_dynamic
+### Case7: test_flush_all
+
+### Testing Objective <!-- omit in toc -->
+Verify flushing of static/dynamic entries on VLAN/Port/All.
+### Test steps: <!-- omit in toc --> 
+- test_flush_vlan_static
+- test_flush_port_static
+- test_flush_all_static
+
+1. Flush with conditions for each case: ``Static`` flush on ``VLAN10``; ``Static`` flush on ``Port1``; flush for all ``Static`` 
+2. Send packets for each case in sequence: ``port1`` DMAC=``Port2 MAC``; ``Port2`` DMAC=``Port1 MAC``; ``port1`` DMAC=``Port2 MAC``
+3. Verify flooding happened, packets received in related VLAN, except the ingress port.
+4. Send packets for each case in sequence:  ``Port9`` DMAC=``Port10 MAC``; ``Port10`` DMAC=``Port9 MAC``; `  ``Port9`` DMAC=``Port10 MAC``
+5. Verify flush happens in a certain domain, unicast to the corresponding port.
```

Comment:
should we add two cases? for flush, it should take effect on some vlan or port, for those ones, we should expect flood. for which is not in the flush range, it should be unicast. @richardyu-ms

Reply:
yes, it should be better to split them into different cases